gerrymandering/gerrymandering.ipynb

```diff
@@ -116,17 +110,30 @@
    "outputs": [],
    "source": [
     "x_range, y_range = ((-13884029.0, -7453303.5), (2818291.5, 6335972.0)) # Continental USA\n",
-    "shade_defaults = dict(x_range=x_range, y_range=y_range, x_sampling=10, y_sampling=10, width=1200, height=682,\n",
-    "                      color_key=color_key, aggregator=ds.count_cat('race'),)\n",
-    "shaded = datashade(census_points, **shade_defaults)\n",
+    "\n",
+    "shaded = df.hvplot.points(\n",
+    "    'easting', 'northing',\n",
+    "    datashade=True,\n",
+    "    aggregator=ds.count_cat('race'),\n",
+    "    cmap=color_key,\n",
+    "    xlim=x_range,\n",
+    "    ylim=y_range,\n",
+    "    dynspread=True,\n",
+    "    height=800,\n",
+    "    width=1000,\n",
+    "    data_aspect=1,\n",
+    "    tiles='CartoLight',\n",
+    "    rasterize=True\n",
```

Comment:
```suggestion
```

Seems like this line needs to be deleted if `datashade=True` above.

Reply:
Change made!